packages/build/tests/core/snapshots/tests.js.md

```diff
@@ -81,6 +81,7 @@ Generated by [AVA](https://ava.li).
           "commandOrigin": "config",␊
           "environment": {}␊
         },␊
+        "functions": {},␊
         "plugins": []␊
       },␊
       "configPath": "/file/path",␊
```

Comment:
Note about `*.snap` files: when using `ava -u` without specifying a file path, those are all updated, even when unchanged. This is due to [a bug in Ava](https://github.com/avajs/ava/issues/2311) which has been fixed, but we cannot upgrade Ava due to supporting Node 8.

In order to prevent merge conflicts with other PRs that do update those test snapshots, we can avoid committing test snapshots that got updated due to that bug (until we finally upgrade Ava).

Reply:
Oh, snap! 🥁 

I didn't know that. Thanks for the tip!